tests/floating-point/fft/fft-cooley-tukey.fil

```diff
@@ -0,0 +1,123 @@
+import "primitives/float.fil";
```

Comment:
Did you mean to commit this file? The components don't seem to be implemented yet. I assuming this is for the follow up PR with the pipelined FFT.